.github/buildomat/jobs/build-and-test.sh

```diff
@@ -21,7 +21,8 @@ banner test
 ptime -m cargo test --lib --verbose
 
 banner test_up.sh
-ptime -m ./tools/test_up.sh
+ptime -m ./tools/test_up.sh unencrypted
+ptime -m ./tools/test_up.sh encrypted
```

Comment:
I always imagined crucible being used to serve ISOs or base OS images or something like that in the short term, meaning we have to support both code paths for now. Maybe that's not true.

Reply:
Yeah, I'm not ready to get rid of the unencrypted path yet either.  Both because we might want it and because there will be times when debugging something is easier with it off.